client/service-worker.js

```diff
@@ -32,8 +32,8 @@ function showNotification(event, payload) {
 
 				return self.registration.showNotification(payload.title, {
 					tag: `chan-${payload.chanId}`,
-					badge: "img/logo-64.png",
-					icon: "img/touch-icon-192x192.png",
+					badge: "img/favicon-alerted-transparent-bg.svg",
+					icon: "img/favicon-grey-bg.svg", // TODO: Obtain and switch to logo-grey-bg.svg (2-hex)
```

Comment:
svgs dont work in Chrome and Edge

Reply:
Like, at all? Only these 2 things need to go from SVG to PNGs, or somewhere else as well?

Reply:
Both don't work yeah. Also icons in non-SW notifications.